cineast-core/src/main/java/org/vitrivr/cineast/core/util/ocr/ThreadLocalObjectCache.java

```diff
@@ -1,4 +1,4 @@
-package org.vitrivr.cineast.core.util;
+package org.vitrivr.cineast.core.util.ocr;
```

Comment:
This one is not specifically something for OCR, even though it is currently only used there.